makefile

```diff
@@ -182,8 +182,8 @@ ifeq ($(MFEM_USE_OPENMP),YES)
 endif
 
 # List of MFEM dependencies, processed below
-MFEM_DEPENDENCIES = METIS LIBUNWIND SIDRE LAPACK OPENMP SUNDIALS MESQUITE\
- SUITESPARSE SUPERLU STRUMPACK GECKO GNUTLS NETCDF PETSC MPFR
+MFEM_DEPENDENCIES = SUPERLU METIS LIBUNWIND SIDRE LAPACK OPENMP SUNDIALS MESQUITE\
```

Comment:
Any reason the order was changed here?

Reply:
Yes. At least on the lab linux clusters using intel and clang compilers, superlu was missing some metis calls in the linking step due to it being last in the linking list.